src/components/mindset/ReadMore.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+// Material UI
+import Link from '@material-ui/core/Link'
+
+
+const ReadMore = () => {
+  return (
+    <Link href="/" color="inherit" underline="always">Read more about civil tort cases in Arizona</Link>
+  )
+}
+
+export default ReadMore
```

Comment:
I'm _positive_ this isn't the right href for Read More, but I don't have a route or a page for that yet.

Reply:
Also, this is one of the places where I don't know where I'm grabbing state-specific content from.

Reply:
FYI on this too - if we're staying in the application, then we'll want to use the `react-router` `Link` component. I actually called the MaterialUI Link `MaterialLink` in some places, just to avoid the two Link situation. You should be able to handle this similar to the button, where you add `component={Link} to="/read-more-about-this-thing"` once we know where that url will go :)